app.js

```diff
@@ -18,40 +19,55 @@ go();
 async function go() {
   try {
     for (const org of config.orgs) {
+      console.log('⏳ Collecting repos for', org)
       let page = 1;
       while (true) {
         const params = {
           page,
-          access_token: config.token
+          sort
         };
         const url = `https://api.github.com/orgs/${org}/repos?${qs.stringify(params)}`;
+
         const repos = await request(url, {
           json: true,
           headers: {
-            'User-Agent': 'changelog-scanner'
+            'User-Agent': 'changelog-scanner',
+            'Authorization': `token ${config.token}`
           },
         });
+
+        if (!Array.isArray(repos)) {
+          console.error('Error retrieving repos:', repos);
```

Comment:
It came through as a stringified partial array a few times. Maybe this shouldn't include `repos` since in that case it's not super usable information.